caffe2/mobile/contrib/ios/mpscnn/mpscnn.mm

```diff
@@ -256,12 +256,12 @@ void computeOutputHW(
     int W,
     int* OH,
     int* OW) {
-  Tensor input(CPU), output(CPU);
-  input.Resize(1, 1, H, W);
+  std::vector<int64_t> dims = {1, 1, H, W};
```

Comment:
I don't know if the constructor can handle initializer lists, but I know that empty definitely can:

```
Tensor input = empty({1, 1, H, W});
```

No need to allocate an entire vector here.

Reply:
looks like Tensor here is just used for record sizes, no actual memory is allocated...